test/execution-test-data/flowtriggeryamltest/flow_trigger.flow

```diff
@@ -0,0 +1,58 @@
+---
+# Flow trigger
+trigger:
+  maxWaitMins: 5
+  schedule:
+    type: cron
+    value: 0 0 1 ? * *
+
+  triggerDependencies:
```

Comment:
can we handle the case of no triggerDependencies?

Reply:
Yes I can add the support for no triggerDependencies. Also do we need to support no params case?